test/Connector.t.sol

```diff
@@ -79,7 +79,7 @@ contract ConnectorTest is Test {
     }
 
     function testUpdatingMemberWorks(uint64 poolId, bytes16 trancheId, address user, uint256 validUntil) public {
-        vm.assume(validUntil > block.timestamp);
+        vm.assume(validUntil > safeAdd(block.timestamp, new Memberlist().minimumDelay()));
```

Comment:
We should add specific tests `testUpdatingMemberBeforeMinimumDelayFails` that check that updating a member to `< now + minimumDelay` reverts.

Reply:
Good idea

Reply:
https://github.com/centrifuge/connectors/issues/20